### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by kg345**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #ECCN: Open Source
-#GUSINFO: Education Data Architecture (EDA), Education Data Architecture (EDA)
-
+#GUSINFO: Rad-ish, Education Data Architecture & K-12 (MP)
```
Comment (robertbwatson) [inline CODEOWNERS:2]: Are spaces okay? The other codeowners PRs don't have spaces, so just want to make sure this is fine

Comment (kg345) [reply CODEOWNERS:2]: It's a good question - I was going off the confluence page where the examples show spaces in the names of product tag, at least.
https://confluence.internal.salesforce.com/display/corescm/Create+a+GUS-Aware+CODEOWNERS+File

**Comment by kg345** [comment]
Closing + opened a new PR with correct branch naming
https://github.com/SalesforceFoundation/EDA/pull/1996

File: CODEOWNERS (hunk 1 above)
Comment (robertbwatson) [reply CODEOWNERS:2]: Okay. I just didn't know if "#GUSINFO: Rad-ish, Education Data Architecture & K-12 (MP)" vs "#GUSINFO:Rad-ish,Education Data Architecture & K-12 (MP)" made a difference, as I noticed all of the others are the latter.

Comment (kg345) [reply CODEOWNERS:2]: It seems they [trim the line ](https://git.soma.salesforce.com/scm/codeowners_parser/blob/master/src/main/java/com/salesforce/codeowners/parser/CodeownerParser.java#L314) when the split the string (based on `#GUSINFO`) so I believe it should be ok with a space.

**Pull request closed without merging**